server/src/utils/chess-game-util.js

```diff
@@ -0,0 +1,56 @@
+const calcExpectedScore = (_ratingA, _ratingB) => {
+  const ratingA = parseInt(_ratingA, 10);
+  const ratingB = parseInt(_ratingB, 10);
+  const score = 1 / (1 + (10 ** ((ratingB - ratingA) / 400)));
+  return parseFloat(score.toFixed(2));
+};
+
+const calcScores = (winner, pOneExpScore, pTwoExpScore) => {
+  let pOneScore;
+  let pTwoScore;
+
+  switch (winner) {
+    case 'player_one':
+      pOneScore = 1;
+      pTwoScore = 0;
+      break;
+    case 'player_two':
+      pOneScore = 0;
+      pTwoScore = 1;
+      break;
+    case 'draw':
+      pOneScore = 0.5;
+      pTwoScore = 0.5;
+      break;
+    default:
```

Comment:
This case must be reported as there can only be `1/0 0/1 0.5/0.5`, may be it could be checked at higher layer and this is never called if there is 4th case.